prod/prod.go

```diff
@@ -18,43 +18,109 @@ package prod
 
 import (
 	"context"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"time"
 
+	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/ethersphere/swarm/chunk"
 	"github.com/ethersphere/swarm/pss"
 	"github.com/ethersphere/swarm/pss/trojan"
+	"github.com/ethersphere/swarm/storage/feed"
+	"github.com/ethersphere/swarm/storage/feed/lookup"
 )
 
-// RecoveryHook defines code to be executed upon trigger of failed to be retrieved chunks
+// RecoveryTopicText is the string used to construct the recovery topics
+const RecoveryTopicText = "RECOVERY"
+
+// RecoveryTopic is the topic used for repairing globally pinned chunks
+var RecoveryTopic = trojan.NewTopic("RECOVERY")
```

Comment:
here use RecoveryTopicText

Reply:
whoops, this was left behind when i split the code into 2 more PRs.

fixed in the main branch, thanks